app/src/main/java/org/xwiki/android/sync/bean/XWikiUserFull.java

```diff
@@ -100,8 +117,12 @@ public void setPhone(String phone) {
         setValue("phone", phone);
     }
 
+    /**
+     * @return Avatar URL without base url
+     */
     public String getAvatar() {
-        return searchValue("avatar");
+        return "bin/download/XWiki/" + pageName
```

Comment:
I don't like harcoding this URL so much. At the very least the space of the user profile ("XWiki" most of the time but could be something else) should be remembered with the page name.